packages/v4/src/content/design-guidelines/content/capitalization.md

```diff
@@ -5,29 +5,78 @@ hideTOC: true
+
+Each company handles UI capitalization differently, so align with your own company’s rules when designing your UIs. If you’re designing UIs for Red Hat products, follow these rules: 
+
+* Write menu/navigation items and their corresponding page titles in **title case**. 
+<br />
+* Write products, services, and features in **title case**.
+<br />
+* Write all other areas (like button text, form field labels, and empty state headings) in **sentence case**. 
 
-Example:
-<br />Rules > Systems > vagrant-host
+Capitalize **Red Hat feature names** when they’re used as proper nouns or when they refer to a capitalized UI term (like a navigation item). Write them in lowercase when they’re used to describe generic concepts.
 
-You always want the names in the breadcrumb trail to exactly match the names of the elements they’re referencing.
+Examples: 
+<table style="table-layout: fixed" tr width="80%">
+    <tr>
+        <th><center><strong>Feature&nbsp;name</strong></center></th>
+        <th><center><strong>UI text</strong></center></th>
+        <th><center><strong>Reasoning</strong></center></th>
+    </tr>
+    <tr>
+        <td><center>Compliance</center></td>
+        <td>“Check your system compliance using Red Hat Insights Compliance.”</td>
+        <td>The first “compliance” is lowercase since it refers to compliance as a concept. <br /><br />The second “compliance” refers to a specific feature offered on cloud.redhat.com.
+</td>
+</tr> 
+<tr>
+         <td><center>User Access</center></td>
+	        <td>“Manage user access for your organization using the User Access feature.”</td>
+            <td>The first “user access” is lowercase because it refers to user access as a concept. <br /><br />The second “user access” is capitalized because it refers to the User Access feature offered on cloud.redhat.com.
+</td>
+</tr>
+    <tr>
+        <td><center>Sources</center></td>
+        <td>"Add a source by going to <b>Settings</b> > <b>Sources</b>." <br /><br />"Check the <b>Sources</b> table for status." <br /><br />Button text: "Add source"
+</td>
+        <td>“Sources” is only capitalized when it directly refers to a subsection, feature, or location in the UI.<br /><br />Always write buttons in sentence case.
+</td>
+</tr>
+</table>
 
-## Capitalization for PatternFly components
-PatternFly components should be written in lowercase. Components are not technically proper nouns or products, so they aren’t capitalized. If they start a sentence or a page title, then capitalize the first letter only.
+##Capitalization for tools outside your product portfolio
+If you’re referencing tools outside your company’s product portfolio, write the product names as they appear in the respective company’s documentation. 
+
+For example, if you’re referencing a product in AWS and Amazon capitalizes it, capitalize it in your writing.
+
+##Capitalization for breadcrumb trails
```

Comment:
Same thing here, gotta add a space between the hashtags and the header, and add an "enter" between the header and the body. (essentially like adding a page break, but you don't need to add the br code, just press enter again after the header so there is visually vertical space between the header and the sentence)